app/components/AddOrganisationModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {useState} from 'react';
+import {Button, Modal} from 'react-bootstrap';
+import JsonSchemaForm, {IChangeEvent} from 'react-jsonschema-form';
+import FormFieldTemplate from 'containers/Forms/FormFieldTemplate';
+import FormObjectFieldTemplate from 'containers/Forms/FormObjectFieldTemplate';
+import addOperatorSchema from 'components/organisation/addOrganisationSchema';
+
+interface Props {
+  onAddOrganisation?: (...args: any[]) => void;
+}
+
+const AddOrganisationModal: React.FunctionComponent<Props> = (props) => {
+  const {onAddOrganisation} = props;
+  const [isModalVisible, setModalVisible] = useState(false);
+
+  const customFormats = {
+    'postal-code': /[a-z]\d[a-z]\s?\d[a-z]\d/i,
+    duns: /^\d{9}$/
```

Comment:
The application form does not collect duns number anymore. So we may want something else in this view. CRA number maybe? Maybe we can get a quick clarification on what to do here (or is that a leftover from a previous version of the form)

Reply:
yeah I did a lot of copy/paste moving stuff around from existing code, this is probably just a leftover